entries/b/buildium.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Buildium": {
+    "domain": "buildium.com",
```

```suggestion
    "domain": "buildium.com",
    "additional-domains": [
      "managebuilding.com"
    ],
```
It seems that they have a login page available at this domain